### ISSUE TITLE
Onchange not triggered first time with keyboard

### ISSUE THREAD

**Issue by pranayrauthu** [benbowes/react-responsive-select#106]
onchange event is not triggered first time if options are selected with keyboard.

![ezgif-5-16d462aeb16e](https://user-images.githubusercontent.com/22414856/49499358-cb549880-f893-11e8-9520-bbb4ce584391.gif)

**Comment by benbowes** [comment]
I will look into this @pranayrauthu

**Comment by yyaswanth** [comment]
@benbowes Any ETA for this issue.

**Comment by benbowes** [comment]
I'm on holiday atm @yyaswanth, I plan to give this time when I get back in mid Jan.

**Comment by benbowes** [comment]
I don't believe that the onchange event should be triggered the first time. The reason is that the value has not changed. 

Let me know if I have not understood your actual issue.
